pay-stamp-duty-land-tax/v1-diff.html

```diff
@@ -2,6 +2,7 @@
 layout: pay-stamp-duty-land-tax
 title: Pay Stamp Duty Land Tax
 version: 1
+published_date: 2 December 2015
```

Comment:
Rather than needing this data here you can use the infinitely more verbose `site.data.pay-stamp-duty-land-tax.published` in your template instead of `page.published_date`. Then pipe it through the [date filter](https://shopify.github.io/liquid/filters/date/) with `'%-d %B %Y'`